### PR TITLE
Bug catch, Feature 'chat' development

### PR DESCRIPTION
**Pull request by Jbrooks00**
Im attempting to merge this fixes a null error ive been having, as well as implements the chat title, name, user selection list.

### CODE REVIEW

**Comment by Jbrooks00** [comment]
App module merge conflict, just some formatting issues deleted. please review

File: src/app/core/services/chat.service.ts [hunk 1]
```diff
@@ -13,12 +13,14 @@ import {
   where,
```
Comment (Anth0nyQ) [inline src/app/core/services/chat.service.ts:13]: Good work, Joseph, progress on the chat feature is looking good!

**Review by Anth0nyQ — APPROVED** [review]
Code looks clean!

**Comment by Jbrooks00** [comment]
I had to force push Hash 'bb...' to 'b7...' under the assumption code was merged into branch 'developmental' had to reset to previous commit.

Could someone please review as well as accept the merge if no issues.

**Pull request merged**